pkgs/development/python-modules/curl-cffi/default.nix

```diff
@@ -79,8 +99,11 @@ buildPythonPackage rec {
   disabledTestPaths = [
     # test accesses network
     "tests/unittest/test_smoke.py::test_async"
+    # Hangs the build (possibly forever) under websockets > 12
+    # https://github.com/lexiforest/curl_cffi/issues/657
+    "tests/unittest/test_websockets.py/test_websocket"
```

Comment:
```suggestion
    "tests/unittest/test_websockets.py::test_websocket"
```

Reply:
Yeah, that was a mistake. Fixed and tested with nix-build.